src/store/proposals/proposals.slice.ts

```diff
@@ -67,26 +67,23 @@ export const appSlice = createSlice({
     setGovernor: (state, { payload }: PayloadAction<string>) => {
       state.selectedGovernor = payload;
     },
-    watchAddProposal: (state) => {
-      state.addProposalEligible = 'loading';
-    },
+    watchAddProposal: () => {},
     stopWatchingAddProposal: (state) => {
       state.addProposalState = 'idle';
     },
-    checkAddPropsal: (state) => {
-      state.addProposalEligible = 'loading';
+    checkAddProposal: (state) => {
+      state.reasonToBlockProposal = undefined;
```

Comment:
Maybe let's remove clearing the state here so it wont clear and populate on each block